package.json

```diff
@@ -21,7 +21,7 @@
     "@babel/plugin-proposal-async-generator-functions": "7.2.0",
     "@babel/preset-react": "7.0.0",
     "acorn": "6.1.1",
-    "babel-eslint": "10.0.1",
+    "babel-eslint": "10.0.3",
```

Comment:
https://docs.google.com/document/d/1Oguqp43W4_ChyroJ9AJAzG1jSwkUWfKvBKVvrDxVsMg/edit#bookmark=id.81i7mkprevgg makes me think that for each of the packages that we're changing in this PR, we're going to have to check those parts of the dependency tree and mention that in the bug. The simplest way to do this at the moment would be to put the packages into broofa, like [this](http://npm.broofa.com/?q=eslint@6.2.2) to verify that those licenses match the runbook policies (yeah, we'll want to automate this soon, and I'll be filing a bug for that).

If @Standard8 beats us to the punch and annotates https://bugzilla.mozilla.org/show_bug.cgi?id=1540982 before you get to this, we can presumably just state that this has already been taken care of for mozilla-central.

Reply:
eslint@5.16.0:
Apache-2.0(4)
BSD-2-Clause(7)
BSD-3-Clause(3)
ISC(15)
MIT(82)
WTFPL OR MIT

and upgrading to eslint@6.2.2:
Apache-2.0(4)
BSD-2-Clause(7)
BSD-3-Clause(3)
ISC(16)
MIT(85)

so fewer types of licenses, so I guess we're okay here for eslint (and that m-c uses eslint anyway)

---

babel-eslint@10.0.1:
Apache-2.0
BSD-2-Clause(4)
BSD-3-Clause
MIT(26)

babel-eslint@10.0.3:
Apache-2.0
BSD-2-Clause
BSD-3-Clause
MIT(28)

so same set

---

adding eslint-plugin-html@6.0.0:
BSD-2-Clause(4)
ISC(2)
MIT(6)

subset of allowed licenses

---

eslint-plugin-mozilla@1.3.0:
MPL-2.0
Unspecified
BSD-2-Clause(4)
ISC(2)
MIT(6)

to eslint-plugin-mozilla@2.1.0:
MPL-2.0
Unspecified
BSD-2-Clause(4)
ISC(2)
MIT(6)

same set… including null/undefined but I suppose this plugin is from mozilla-central anyway

Reply:
Resolved as OK after discussion in the bugzilla bug.